tests/dragonfly/replication_test.py

```diff
@@ -1897,11 +1897,11 @@ async def test_replicaof_reject_on_load(df_local_factory, df_seeder_factory):
     replica = df_local_factory.create(dbfilename=f"dump_{tmp_file_name}")
     df_local_factory.start_all([master, replica])
 
-    seeder = df_seeder_factory.create(port=replica.port, keys=30000)
+    seeder = df_seeder_factory.create(port=replica.port, keys=40000)
```

Comment:
I would add another print line below 1911 (replica of command)
assert "loading:0" in (await async_client.execute_command("INFO PERSISTENCE"))
this way we will know if the server finished loading and add there a comment that if this happens we need to increase the keys.

Reply:
you mean before `replicaof` command not after, because after we will be in loading state since replication has started. It's a good idea to add `assert loading:1` before the `replicaof` command but keep in mind this is again a heurestic, it might be the case that snapshot is loading when we assert but on the next statement `replicaof` the snapshot is done and `replicaof` succeeds. But it's an extra guard so +1